docs/scripting-ref.rst

```diff
@@ -2230,3 +2230,128 @@ Trigger mouse release event in web page.
 **Async:**: no.
 
 See notes about mouse events in :ref:`splash-mouse-click`.
+
+.. _splash-send-keys:
+
+splash:send_keys
+----------------
+
+Send keyboard events to page context.
+
+**Signature:** ``splash:send_keys(keys)``
+
+**Parameters**
+
+* keys - string representing the keys to be sent as keyboard events.
+
+**Returns:** nil
+
+**Async:**: no.
+
+Key sequences are specified by using a small subset of emacs edmacro syntax:
+
+* whitespace is ignored and only used to separate the different keys
+* characters are literally represented
+* ``RET``, ``SPC``, ``TAB``, ``DEL``, ``ESC`` are reserved and represent
+  special control characters.
+* words within brackets represent function keys, like ``<Return>``, ``<Left>``
+  or ``<Home>``. See `Qt docs`__ for a full list of function keys. ``<Foo>``
+  will try to match ``Qt::Key_Foo``.
```

Comment:
None, from https://github.com/emacs-mirror/emacs/blob/master/lisp/edmacro.el

```

 * A word in angle brackets, e.g., <return>, <down>, or <f1>, represents
   a function key.  (Note that in the standard configuration, the
   function key <return> and the control key RET are synonymous.)
   You can use angle brackets on the words RET, SPC, etc., but they
   are not required there.
```

Maybe we prefer non-capitalized words for function keys? <return> and <down> instead of <Return> and <Down>?